app/components/Image.js

```diff
@@ -22,7 +22,10 @@ export default class Image extends Component {
     onMount: PropTypes.func,
     onLoad:  PropTypes.func,
     onError: PropTypes.func,
-    style:   PropTypes.array
+    style:   PropTypes.oneOfType([
+      PropTypes.array,
```

Comment:
does array actually work here?  it would mean our `style=` has a nested array; maybe radium flattens arrays out?